scripts/bump-marshal.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Bump "marshal" version in common/
+#
+# Usage:
+#     bump-marshal.sh MARSHAL_GIT_SHA [--no-commit]
+#
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $SCRIPT_DIR/..
+
+COMMIT="1"
+
+while [[ $# -gt 0 ]]; do
+    case "${1}" in
+    -n|--no-commit)
+        COMMIT=""
+        shift
+        ;;
+    *)
+        MARSHAL_SHA="${1}"
+        shift
+        ;;
+    esac
+done
+
+sed -i '' -e "s|^\(marshal.*github.com/getsentry/marshal\", rev = \)\"[a-z0-9]*\"|\1\"${MARSHAL_SHA}\"|" \
+    common/Cargo.toml
+
+cargo update -p semaphore
+
+cd cabi
+cargo update -p semaphore-common
+
+if [ -n "${COMMIT}" ]; then
+    # Commit changes
+    git commit --all -n -m "getsentry/marshal@{sha}"
```

Comment:
Commit message should have a prefix, e.g. `"build: Bump getsentry/marshal@${MARSHAL_SHA}"`